operationalGuide.md

```diff
@@ -0,0 +1,174 @@
+# Operational Guide
+
+- Scope of the Card API | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Scope-of-the-Card-API)
+- Central Design Decisions | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Central-Design-Decisions)
+- Card API Level 1 | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Card-API-Level-1)
+- Card API Level 2 | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Card-API-Level-2)
+- Card API Level 3 | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Card-API-Level-3)
+- Use of this spec | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Use-of-this-spec)
+- Appendix | [Card Wiki Page](https://github.com/swissfintechinnovations/ca-card/wiki/Appendix)
+
+# Scope of the Card API
+
+## Card types covered with the Card API
+In scope:
+* Debit
+* Credit
+* Prepaid
+
+Ouf of scope:
+* Deposit cards or non-Scheme-based cards (e.g.loyalty cards) 
```

Comment:
Why are these cards out of scope? The idea should be that bank clients can see all their cards

Reply:
We focus in a first step on the mentioned most commonly used cards (MVP approach).
Are you missing other cards than those listed explicitly in the out of scope?

Reply:
In our opinion, Deposit cards are part of MVP, becasuse that ist a common product. I also recommend to scopee Loyality cards (we have banks with such cards)

Reply:
Deposit cards will be added as card type in the enum. It is left to data providers if they want to provide these type of cards.
Definition of loyalty cards will be refined to clarify that such with scheme-based payment functionality are included.